src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -0,0 +1,166 @@
+                    break;
+                }
+                case CONST.POLICY.CONNECTIONS.NAME.XERO: {
+                    text = translate('workspace.accounting.xero');
+                    onboardingIcon = {
+                        icon: Expensicons.XeroCircle,
+                        iconWidth: 40,
+                        iconHeight: 40,
+                    };
+                    break;
+                }
+                case CONST.POLICY.CONNECTIONS.NAME.NETSUITE: {
+                    text = translate('workspace.accounting.netsuite');
+                    onboardingIcon = {
+                        icon: Expensicons.NetSuiteSquare,
+                        iconWidth: 40,
+                        iconHeight: 40,
+                        iconStyles: [StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR)],
+                    };
+                    break;
+                }
+                default: {
+                    text = translate('workspace.accounting.intacct');
+                    onboardingIcon = {
+                        icon: Expensicons.IntacctSquare,
+                        iconWidth: 40,
+                        iconHeight: 40,
+                        iconStyles: [StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR)],
+                    };
+                    break;
+                }
+            }
+            return {
+                keyForList: connectionName,
+                text,
+                onboardingIcon,
+                isSelected: userReportedIntegration === connectionName,
+            };
+        });
+        const noneAccountingOption: OnboardingListItemData = {
+            text: translate('onboarding.accounting.noneOfAbove'),
+            keyForList: null,
+            onboardingIcon: {
+                icon: Expensicons.Clear,
+                iconFill: theme.success,
+            },
+            isSelected: userReportedIntegration === null,
+        };
+        return [...policyAccountingOptions, noneAccountingOption];
+    }, [StyleUtils, theme.success, translate, userReportedIntegration]);
+
+    const footerContent = (
+        <>
+            {!!error && (
+                <FormHelpMessage
+                    style={[styles.ph1, styles.mb2]}
+                    isError
+                    message={error}
+                />
+            )}
+            <Button
+                success
+                large
+                text={translate('common.confirm')}
+                // eslint-disable-next-line rulesdir/prefer-early-return
+                onPress={() => {
+                    if (!userReportedIntegration) {
+                        setError(translate('onboarding.purpose.errorSelection'));
+                        // eslint-disable-next-line no-useless-return
+                        return;
+                    }
+
+                    // TODO: call CompleOnboarding API after the back-end PR is complete
```

Comment:
To clarify, in this comment, I am asking you to [not block on the backend changes](https://github.com/Expensify/App/issues/48745#issuecomment-2347232176). We still must call the API here and send the relevant params:

```
//    [policyID], - Optional policyID that might exist if the user selected to manage their team expenses.
//    [companySize], - The range of employee headcount. Options: "1-10", "11-50", "51-100", "101-1000", "1001+".
//    [userReportedIntegration] - The name of the accounting integration a user told us they use (this is distinct from what they actually have set up).
```

Then we can create a new PR with the task whenever the copy is ready. Let me know if it's clear now! Thanks!

Reply:
```
if (!onboardingPolicyID) {
    const {adminsChatReportID, policyID} = Policy.createWorkspace(undefined, true, work);
    Welcome.setOnboardingAdminsChatReportID(adminsChatReportID);
    Welcome.setOnboardingPolicyID(policyID);
} else {
    Policy.updateGeneralSettings(onboardingPolicyID, work);
}
```

@marcaaron In the past we will call this logic to create a new policy in the onboarding work step. For now, we will only generate a policyID then create optimistic policy data in `compleOnboarding` or we will call the same logic above and only pass `policyID` to `compleOnboarding` function.

Reply:
I didn't entirely understand what you said there. But let's call `CreateWorkspace` it as soon as you select "Manage my team's expenses" then?

Reply:
Got it.

Reply:
> Since we are not going to use the policy in between steps is there any reason that we create the workspace at that early step? Why not create it just before calling completeOnboarding in the accounting step?

@s77rt The context here.